tests/test_crawl_manager.py

```diff
@@ -169,6 +170,7 @@ def _test_limit_runtime(self):
         self.crawl_manager.limit_runtime(self.spider)
         self.assertTrue(self.crawler.engine.close_spider.called)
 
+    @pytest.skip("why is it failing?")
```

Comment:
it's a bug in test case :) https://github.com/scrapinghub/scrapyrt/pull/38/files#diff-54bd5763ec169d01e8aa9c17fde9d1b9R164 - change to datetime.datetime.utcnow()